src/ui/api/user.go

```diff
@@ -116,6 +116,7 @@ func (ua *UserAPI) Get() {
 			ua.CustomAbort(http.StatusInternalServerError, "Internal error.")
 		}
 		u.Password = ""
+		u.HasAdminRole = ua.SecurityCtx.IsSysAdmin()
```

Comment:
This is incorrect, it does not cover the case when the user id of the api is not current user's

Reply:
fixed it